docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -106,6 +106,30 @@ export S3_ENDPOINT_URL='https://<accountid>.r2.cloudflarestorage.com'
 ```
 ````
 
+One can also set credentials through environment variables. For example, with Backblaze:
+
+```{note}
+Your endpoint url is 'https://s3.<your region>.backblazeb2.com'. The account ID can be retrieved through your [Backblaze console](https://secure.backblaze.com/b2_buckets.htm).
```

Comment:
Can you please combine this line with line 95?

Reply:
Done